state/etcd/etcd.go

```diff
@@ -134,7 +138,10 @@ func (e *Etcd) Features() []state.Feature {
 }
 
 func metadataToConfig(connInfo map[string]string) (*etcdConfig, error) {
-	m := &etcdConfig{}
+	m := &etcdConfig{
+		// This is the default value for maximum ops per transaction, configurtable via etcd server flag --max-txn-ops.
+		MaxTxnOps: 128,
```

Comment:
Can you make this a constant?

Reply:
I guess - but this right here is how we initialize defaults other places too. I can point you to many examples in contrib.

Reply:
Ok. It was just another nit too.